android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -134,7 +137,7 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
 	@Override
 	public void onCreate(Bundle savedInstanceState)
 	{
-		setFullscreen(true);
+		setFullscreenForCamera(true);
```

Comment:
Since base class should be agnostic of derived classes, adding camera specific method in base class is not advised. 

`getWindow().addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);` can be moved here. Also this is being used specifically for TiCameraActivity only.